lib/replicator.js

```diff
@@ -2090,7 +2093,8 @@ module.exports = class Replicator {
 
     if (channel === null) return onnochannel()
 
-    const peer = new Peer(replicator, protomux, channel, session)
+    const opts = { inflightRange: this.inflightRange }
+    const peer = new Peer(replicator, protomux, channel, session, opts)
```

Comment:
this is an internal object so just pass the inflightRange as an argument. its not optional here in practice